api/capi/include/nnstreamer.h

```diff
@@ -885,6 +904,43 @@ int ml_pipeline_element_get_property_double (ml_pipeline_element_h elem_h, const
  */
 int ml_pipeline_element_get_property_enum (ml_pipeline_element_h elem_h, const char *property_name, uint32_t *value);
 
+/****************************************************
+ ** NNStreamer Pipeline tensor_if Control          **
+ ****************************************************/
+/**
+ * @brief Registers a tensor_if custom callback.
+ * @details If the if-condition is complex and cannot be expressed with tensor_if expressions, you can define custom condition.
+ * @since_tizen 6.5
+ * @remarks If the function succeeds, @a tensor_if handle must be released using ml_pipeline_if_custom_unregister().
+ * @param[in] name The name of tensor_if
+ * @param[in] cb The function to be called when the pipeline runs.
+ * @param[in] user_data Private data for the callback. This value is passed to the callback when it's invoked.
+ * @param[out] if_custom The tensor_if handler.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER The parameter is invalid, or duplicated name exists.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory to register the custom filter.
+ *
+ * Here is an example of the usage:
+ * @code
+ * TBU
+ * @endcode
+ */
+int ml_pipeline_tensor_if_custom_register (const char *name, ml_pipeline_if_custom_cb cb, void *user_data, ml_pipeline_if_h *if_custom);
```

Comment:
We are registering a custom condition to the process globally. We need to write some notes or warning on this.
(If we register a custom condition "X", this "X" may be referred in any pipelines of the current process)

Reply:
Is it enough to add a comments? or Post debug or warning messages?

Reply:
It is to be commented: let users know that a registered if-custom can be referred anywhere from the user process.